src/stores/contactmoment.ts

```diff
@@ -1,14 +1,41 @@
+import type { Zaak } from "@/features/zaaksysteem/types";
```

Comment:
Besproken: om te voorkomen dat features van elkaar afhankelijk zijn, gebeurt de orkestratie nu in de view. Dat is ok. Voor deze store zou het vervelend kunnen worden als deze voor alle soorten objecten die je kan koppelen, losse lijstjes moet bijhouden. Puur voor de koppeling heb je namelijk alleen een object met een id nodig, de store hoeft in principe niets te weten over de verschillende objecttypes (zaak/klant/notitie/etc). Dat maakt het lastiger om een lijst te tonen van objecten van een specifiek type (bv een zaak). Maar die zou je ook in een eigen store kunnen bijhouden. Hier moeten we goed over nadenken bij de story waarin we klanten gaan koppelen aan een contactmoment.